### DISCUSSION TITLE
Transclude BUFFER instead of FILE?

### DISCUSSION THREAD

**Discussion by devcarbon-com** [nobiot/org-transclusion#162] [Ideas]
Not sure if this is possible even, but I think it would be really cool to be able to transclude a buffer, like say, the last ten lines of the messages buffer, or the agenda buffer.

**Comment by nobiot** [comment]
None of the current add-functions does it but it should be possible. All the transclusions are actually from a buffer content. Would you care to try?

**Reply by devcarbon-com** [reply]
Cool!  Yeah, I'll take a look when I'm able. :)
